### PR TITLE
Log4j 2.16.0 is not sufficient. Upgrading to 2.17.0

### PR DESCRIPTION
**Pull request by MichaelSp**
Upgrade log4j once more due to https://nvd.nist.gov/vuln/detail/CVE-2021-45105

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- ~[ ] Link to relevant issues in GitHub or Jira~
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- ~[ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue~

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]
Log2j 2 2.16.0 probably is sufficient, but it is good to use the most recent update in any case.

See https://community.jenkins.io/t/apache-log4j-2-vulnerability-cve-2021-44228/990/12 for why I believe the issue reported in 2.16.0 is not actually an issue for a Jenkins installation.

**Comment by coxalass** [comment]
Hello,

Following this publication : https://checkmarx.com/blog/cve-2021-44832-apache-log4j-2-17-0-arbitrary-code-execution-via-jdbcappender-datasource-element/

Can you please upgrade to 2.17.1 and produce a release ?

Kind regards,

File: build.gradle [hunk 1]
```diff
@@ -60,9 +60,9 @@ dependencies {
             'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.11.3',
             'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.10.5',
             'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.10.5',
-            'org.apache.logging.log4j:log4j-slf4j-impl:2.16.0',
-            'org.apache.logging.log4j:log4j-api:2.16.0',
-            'org.apache.logging.log4j:log4j-core:2.16.0'
+            'org.apache.logging.log4j:log4j-slf4j-impl:2.17.0',
+            'org.apache.logging.log4j:log4j-api:2.17.0',
+            'org.apache.logging.log4j:log4j-core:2.17.0'
```
Comment (MichaelSp) [inline build.gradle:65]:
```suggestion
            'org.apache.logging.log4j:log4j-slf4j-impl:2.17.1',
            'org.apache.logging.log4j:log4j-api:2.17.1',
            'org.apache.logging.log4j:log4j-core:2.17.1'
```

**Comment by coxalass** [comment]
Hello @MichaelSp,

this has been included in latest release of this plugin : https://github.com/jenkinsci/checkmarx-plugin/releases/tag/Release_2022.1.1

Kind regards,

**Comment by MichaelSp** [comment]
💭 (why even care to open a PR here when they are ignored and the same change is later buried in a totally unrelated PR (https://github.com/jenkinsci/checkmarx-plugin/pull/90) 🤔 strange way to maintain the code....)

**Pull request closed without merging**